docs/2.0/05-filters-en.md

```diff
@@ -10,6 +10,7 @@ Filter defines the column mutation operations on the Input message. It is config
 Currently, DRC supports the following Filter plugins:
 
 - `reject`: Ignores specific source messages.
```

Comment:
Ignores specific source messages. It's a blacklist filter that rejects specific messages.

Reply:
done